@xen-orchestra/web/src/components/hosts/HostsTable.vue

```diff
@@ -0,0 +1,251 @@
+                      :key="tag"
+                      v-tooltip="false"
+                      accent="info"
+                      variant="secondary"
+                    >
+                      {{ tag }}
+                    </UiTag>
+                  </UiTagsList>
+                </div>
+                <div v-if="column.value.length > 2" class="typo-body-regular-small more-info">
+                  {{ `+${column.value.length - 2}` }}
+                </div>
+              </div>
+            </td>
+          </tr>
+        </template>
+      </VtsDataTable>
+      <VtsStateHero v-if="searchQuery && filteredHosts.length === 0" format="table" type="no-result" size="small">
+        {{ t('no-result') }}
+      </VtsStateHero>
+      <UiTopBottomTable :selected-items="0" :total-items="0">
+        <UiTablePagination v-bind="paginationBindings" />
+      </UiTopBottomTable>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useXoHostUtils } from '@/composables/xo-host.composable'
+import { useXoPifCollection } from '@/remote-resources/use-xo-pif-collection'
+import type { XoHost } from '@/types/xo/host.type'
+import type { IconName } from '@core/icons'
+import VtsDataTable from '@core/components/data-table/VtsDataTable.vue'
+import VtsIcon from '@core/components/icon/VtsIcon.vue'
+import VtsObjectIcon from '@core/components/object-icon/VtsObjectIcon.vue'
+import VtsStateHero from '@core/components/state-hero/VtsStateHero.vue'
+import UiButtonIcon from '@core/components/ui/button-icon/UiButtonIcon.vue'
+import UiCheckbox from '@core/components/ui/checkbox/UiCheckbox.vue'
+import UiObjectLink from '@core/components/ui/object-link/UiObjectLink.vue'
+import UiQuerySearchBar from '@core/components/ui/query-search-bar/UiQuerySearchBar.vue'
+import UiTablePagination from '@core/components/ui/table-pagination/UiTablePagination.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import UiTopBottomTable from '@core/components/ui/top-bottom-table/UiTopBottomTable.vue'
+import { usePagination } from '@core/composables/pagination.composable'
+import { useRouteQuery } from '@core/composables/route-query.composable'
+import useMultiSelect from '@core/composables/table/multi-select.composable'
+import { useTable } from '@core/composables/table.composable'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { noop } from '@vueuse/shared'
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { hosts } = defineProps<{
+  hosts: XoHost[]
+  hasError: boolean
+}>()
+
+const hostsId = computed(() => hosts.map(host => host.id))
+
+const { t } = useI18n()
+
+const searchQuery = ref('')
+
+const selectedHostId = useRouteQuery('id')
+const { pifsByHost, arePifsReady: isReady } = useXoPifCollection()
+const { selected, areAllSelected } = useMultiSelect(hostsId)
```

Comment:
```suggestion
const { pifsByHost, arePifsReady } = useXoPifCollection()

const { selected, areAllSelected } = useMultiSelect(hostsId)

const isReady = logicAnd(arePifsReady, hostReady)
```

why did you remove this ? 🤔